nimoy/assertions/mocks.py

```diff
@@ -0,0 +1,17 @@
+class MockAssertions:
+    def assert_mock(self, number_of_invocations, mock, method, *args):
+        if not hasattr(mock, method) and number_of_invocations > 0:
+            raise AssertionError(method + " was never invoked") from None
+
+        mocked_method = getattr(mock, method)
+        if (number_of_invocations >= 0) and (mocked_method.call_count != number_of_invocations):
+            raise AssertionError(
+                method + " was to be invoked " + str(number_of_invocations) + " times but was invoked " + str(
+                    mocked_method.call_count)) from None
+
+        for index, value in enumerate(mocked_method.call_args[0]):
```

Comment:
Consider using `for value, expected in zip(mocked_method.call_args[0], args)` instead of enumerating with an index you do not use.